src/content/translations/pl/desci/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Zdecentralizowana nauka (DeSci)
+description: Przegląd zdecentralizowanej nauki na Ethereum
+lang: pl
+template: use-cases
+emoji: ":microscope:"
+sidebarDepth: 2
+image: ../../../../assets/future_transparent.png
+alt: ""
```

Comment:
we should have an `alt` value

Reply:
I actually think we have a lot of alt tags in places we shouldn't 🤔 Alt tags shouldn't be used on decorative images: https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Mh, interesting, we could remove `alt` from this file then